### PR TITLE
[ICC-257] Update io-functions-commons to the latest version

### PR DESCRIPTION
**Pull request by Garma00**
#### List of Changes

* [Update `io-functions-commons` to `26.0.0`](9cd426294a531483aed12578dd36317fb7b8b66c);
* [Fix unit test;](c6d420ce17812ee8da46e7d4d8ccff869b34f6e8)

#### Motivation and Context

In order to use the `ttl` logic we want to update `io-functions-commons`.

#### How Has This Been Tested?

Unit test

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by gquadrati — APPROVED** [review]

**Review by gquadrati — CHANGES_REQUESTED** [review]
Please, add a check to integration tests to verify that `ttl` has not been written

File: __integrations__/index.test.ts [hunk 1]
```diff
@@ -406,6 +408,33 @@ describe("Create Message", () => {
...
+            TE.orElseW(_ => TE.of(O.none as O.Option<MessageContent>))
+          )
+        ),
+        TE.mapLeft(_ => fail(`Error retrieving message data from Cosmos.`)),
+        TE.map(({ message, status, content }) => {
+          expect(O.isSome(message)).toBeTruthy();
+          expect(O.isSome(status)).toBeTruthy();
+          expect(O.isSome(content)).toBeFalsy();
+          expect(O.some(status)).not.toHaveProperty("ttl");
+          expect(O.some(message)).not.toHaveProperty("ttl");
```
Comment (gquadrati) [inline __integrations__/index.test.ts:434]: `O.some` of an Option<Message> is  Option<Option<Message>>. 
We need to extract the value of the option to check the existance of the `ttl` property.

File: __integrations__/index.test.ts [hunk 2]
```diff
@@ -430,8 +430,8 @@ describe("Create Message", () => {
           expect(O.isSome(message)).toBeTruthy();
           expect(O.isSome(status)).toBeTruthy();
           expect(O.isSome(content)).toBeFalsy();
-          expect(O.some(status)).not.toHaveProperty("ttl");
-          expect(O.some(message)).not.toHaveProperty("ttl");
+          expect(O.getOrElse(undefined)(status)).not.toHaveProperty("ttl");
+          expect(O.getOrElse(undefined)(message)).not.toHaveProperty("ttl");
```
Comment (gquadrati) [inline __integrations__/index.test.ts:434]: `getOrElse` needs a Lazy function
```suggestion
          expect(O.getOrElseW(() => undefined)(status)).not.toHaveProperty("ttl");
          expect(O.getOrElseW(() => undefined)(message)).not.toHaveProperty("ttl");
```

Comment (Garma00) [reply __integrations__/index.test.ts:434]: You are right, mb

**Review by gquadrati — APPROVED** [review]
LGTM

**Pull request merged**